HotelManagerment/com.spring.sample/src/com/spring/sample/controller/RegulationsController/GuestTypesController.java

```diff
@@ -0,0 +1,37 @@
+package com.spring.sample.controller.RegulationsController;
```

Comment:
```suggestion
package com.spring.sample.controller.regulations;
```
ri là đủ nhá
ko cần thêm controller đâu

Reply:
Em đặt vậy cho nó đồng bộ tên với mấy controller không cần tách của các thành viên khác á chị

Reply:
cái này là tên package mà cu